test/org/sagebionetworks/bridge/models/PagedResourceListTest.java

```diff
@@ -20,24 +21,37 @@
 
 public class PagedResourceListTest {
 
+    @SuppressWarnings("deprecation")
     @Test
     public void canSerialize() throws Exception {
         List<AccountSummary> accounts = Lists.newArrayListWithCapacity(2);
         accounts.add(new AccountSummary("firstName1", "lastName1", "email1@email.com", "id", DateTime.now(),
                 AccountStatus.DISABLED, TestConstants.TEST_STUDY));
         accounts.add(new AccountSummary("firstName2", "lastName2", "email2@email.com", "id2", DateTime.now(),
                 AccountStatus.ENABLED, TestConstants.TEST_STUDY));
+
+        DateTime startTime = DateTime.parse("2016-02-03T10:10:10.000-08:00");
+        DateTime endTime = DateTime.parse("2016-02-23T14:14:14.000-08:00");
         
-        PagedResourceList<AccountSummary> page = new PagedResourceList<AccountSummary>(accounts, 2, 100, 123)
-                .withFilter("emailFilter", "filterString");
+        PagedResourceList<AccountSummary> page = new PagedResourceList<AccountSummary>(accounts, 2)
+                .withRequestParam("offsetBy", 123)
+                .withRequestParam("pageSize", 100)
+                .withRequestParam("startTime", startTime)
+                .withRequestParam("endTime", endTime)
+                .withRequestParam("emailFilter", "filterString");
         
         JsonNode node = BridgeObjectMapper.get().valueToTree(page);
-        assertEquals(2, node.get("offsetBy").asInt());
-        assertEquals(123, node.get("total").asInt());
+        assertEquals(123, node.get("offsetBy").asInt());
+        assertEquals(2, node.get("total").asInt());
         assertEquals(100, node.get("pageSize").asInt());
         assertEquals("filterString", node.get("emailFilter").asText());
         assertEquals("PagedResourceList", node.get("type").asText());
```

Comment:
This test class was very redundant. I removed the same tests from a second test that is specifically testing that you can provide a null offsetBy, and removed a final test of this class when returned from DDB-based calls... this wasn't adding anything.